.spelling

```diff
@@ -418,6 +418,7 @@ QSFP-DD
 quiescing
 Rados
 Rados-Gateway
+rbd
```

Comment:
Names like rbd should be enclosed in backticks in the docs, not added to the dictionary, since they aren't really words or product names.

```suggestion
```